contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -140,31 +140,31 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint64 subscriptionId,
     bytes calldata data,
     uint32 callbackGasLimit,
-    uint256 gasPriceGwei
+    uint256 gasPriceWei
```

Comment:
Does this need a corresponding change in tooling?

Reply:
I wouldn't think so, because the variable name is only used within the function.
I believe even if the ABI has the old name as long as the type is the same it will be fine

Reply:
Yes, I will need to fix this in the Functions Toolkit & HH starter kit, but I am already tracking.

Reply:
Oh I see what you meant. Yeah, as Morgan said we have the change planned.